src/bout++.cxx

```diff
@@ -430,7 +430,14 @@ int bout_monitor(Solver *solver, BoutReal t, int iter, int NOUT) {
   // Data used for timing
   static bool first_time = true;
   static BoutReal wall_limit, mpi_start_time; // Keep track of remaining wall time
-
+  
+  static bool stopCheck;       // Check for file, exit if exists?
+  static string stopCheckName; // File checked, whose existence triggers a stop
+  
+#ifdef CHECK
+  int msg_point = msg_stack.push("bout_monitor(%e, %d, %d)", t, iter, NOUT);
```

Comment:
Actually the TRACE macro has already been added for this function so I'm just going to remove this `msg_stack` line.